config.yaml

```diff
@@ -66,106 +66,178 @@ languages:
+        - identifier: customnav_agentnav_docker_autodiscovery
+          name: "Autodiscovery"
+          url: "/agent/docker/autodiscovery/"
+          weight: 240
+          parent: customnav_agentnav_docker
+        - identifier: customnav_agentnav_docker_jmx
+          name: "JMX"
+          url: "/agent/docker/jmx"
+          weight: 250
+          parent: customnav_agentnav_docker
+        - identifier: customnav_agentnav_docker_getting_further
+          name: "Getting Further"
+          url: "/agent/docker/getting_further"
+          weight: 260
+          parent: customnav_agentnav_docker
+
+        # Kubernetes 
+        - identifier: customnav_agentnav_kubernetes
+          name: "Kubernetes"
+          url: "/agent/kubernetes/"
+          weight: 30
           parent: menu_references_agent
+        - identifier: customnav_agentnav_host_installation
+          name: "Host installation"
+          url: "/agent/kubernetes/host_installation/"
+          weight: 310
+          parent: customnav_agentnav_kubernetes
+        - identifier: customnav_agentnav_container_installation
+          name: "Container installation"
+          url: "/agent/kubernetes/container_installation/"
+          weight: 320
+          parent: customnav_agentnav_kubernetes
+        - identifier: customnav_agentnav_event_collection
+          name: "Event collection"
+          url: "/agent/kubernetes/event_collection/"
+          weight: 330
+          parent: customnav_agentnav_kubernetes
         - identifier: customnav_agentnav_autodiscovery
           name: "Autodiscovery"
-          url: "/agent/autodiscovery/"
-          weight: 40
+          url: "/agent/kubernetes/autodiscovery/"
+          weight: 390
+          parent: customnav_agentnav_kubernetes
+
+        # AGENT V6 - Subnav start
+        - identifier: customnav_agentnav_v6
+          name: "Agent V6"
+          url: "/agent/v6/"
+          weight: 50
           parent: menu_references_agent
+        - identifier: customnav_agentnav_v6_log_collection
+          name: "Log collection"
+          url: "/agent/v6/log_collection"
+          weight: 510
+          parent: customnav_agentnav_v6
+        - identifier: customnav_agentnav_v6_trace_collection
+          name: "Trace collection"
+          url: "/agent/v6/trace_collection"
+          weight: 520
+          parent: customnav_agentnav_v6
+        - identifier: customnav_agentnav_v6_process_collection
+          name: "Process collection"
+          url: "/agent/v6/process_collection"
+          weight: 530
+          parent: customnav_agentnav_v6
+        - identifier: customnav_agentnav_v6_developer_guide
+          name: "Developer Guide"
+          url: "https://github.com/DataDog/datadog-agent/tree/master/docs/dev"
+          weight: 599
+          parent: customnav_agentnav_v6
+
+        # AGENT V5 - Subnav start
+        - identifier: customnav_agentnav_v5
+          name: "Agent V5 (deprecated)"
```

Comment:
I don't think we can call v5 deprecated yet.

Also lets be consistent on how we write this.  Agent v5 is used elsewhere in the docs. Here was use "V5"